inc/middleware/nx_verified_telemetry.h

```diff
@@ -15,13 +15,21 @@ extern "C" {
 #include "nx_azure_iot_pnp_client.h"
 
 /* Verified Telemetry Components */
+#include "nx_vt_currentsense_component.h"
 #include "nx_vt_fallcurve_component.h"
 
-#define VT_SIGNATURE_TYPE_FALLCURVE 0x01
+#define VT_SIGNATURE_TYPE_FALLCURVE    0x01
+#define VT_SIGNATURE_TYPE_CURRENTSENSE 0x02
+
+#define VT_RECOMMENDED_BUFFFER_SIZE_BYTES 3500
```

Comment:
Typo in BUFFER.
Also with RECOMMENDED, does this mean they can use a smaller value in some cases? If so, how would they determinate how to use less than recommended?

Reply:
updated to VT_MINIMUM_BUFFFER_SIZE_BYTES

Reply:
BUFFER doesnt have 3 F's :D